docs/developer/concepts/duck-typing.md

```diff
@@ -2,17 +2,29 @@
 
 Knative enables [loose coupling](https://en.wikipedia.org/wiki/Loose_coupling) of its components by using [duck typing](https://en.wikipedia.org/wiki/Duck_typing).
 
-Duck typing means that the compatibility of a resource for use in a Knative system is determined by certain properties being present that can be used to identify the resource control plane shape and behaviors. These properties are based on a set of common definitions for different types of resources, called duck types.
+Duck typing means that the compatibility of a resource for use in a Knative system is determined by
+certain properties that are used to identify the resource control plane shape and behaviors.
+These properties are based on a set of common definitions for different types of resources, called
+duck types.
+<!-- What are the duck types? The properties? The common definitions? The types of resources? -->
```

Comment:
Is this something you need input on before merging the PR?

Reply:
It's okay; I think the sentence is clear enough, on balance.